docs/Dockerfile

```diff
@@ -0,0 +1,17 @@
+# Build:
+# docker build -t ace3mod/jekyll .
+#
+# Run:
+# docker run -p 4000:4000 -it ace3mod/jekyll
+
+FROM starefossen/github-pages:onbuild
+MAINTAINER bux
+
+COPY . /usr/src/app
+WORKDIR /usr/src/app
+
+RUN gem install --no-document \
```

Comment:
Ah, you use the `onbuild` image for this. Probably want to switch to the `latest` instead?